Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:8
+
+RUN yum -y update
+RUN yum install -y git gcc make cmake unzip python3-pip
+
+RUN yum install -y yum-utils device-mapper-persistent-data lvm2
+RUN yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+RUN yum install -y docker-ce-18.09.1
+
+VOLUME /var/lib/docker
+
+ENV PORT=2375
+
+ADD wrapdocker /usr/local/bin/wrapdocker
```

Comment:
Are you create your own kind of DinD? Why you use it?

Reply:
Yes, there is no official centos-based dind image.
Other images found on GitHub didn't work for me.

Reply:
What about my idea with current default DinD?